helm/hpcc/values.yaml

```diff
@@ -263,13 +263,10 @@ roxie:
 
 thor:
 - name: thor
-  numWorkers: 2
-  globalMemorySize: 4096
```

Comment:
NB: have got rid of this.. by default Thor allocates 75% of available memory, so whatever the pod has available, Thor will get 75% of it without this setting.
We may want to revisit, but this setting isn't helpful here.